vcd_cli/role.py

```diff
@@ -51,19 +54,52 @@ def list_roles(ctx):
               '--org',
               'org_name',
               required=False,
-              metavar='<org-name>',
+              metavar='<org_name>',
               help='name of the org')
-def list_rights(ctx, role_name, org_name=None):
+def list_rights(ctx, role_name, org_name):
     try:
         client = ctx.obj['client']
-        if (org_name is not None):
+        if org_name is not None:
             org_href = client.get_org_by_name(org_name).get('href')
         else:
             org_href = ctx.obj['profiles'].get('org_href')
-        org = Org(client, org_href, is_sysadmin(ctx))
+        org = Org(client, org_href)
         role_record = org.get_role(role_name)
         role = Role(client, href=role_record.get('href'))
         rights = role.list_rights()
         stdout(rights, ctx)
     except Exception as e:
         stderr(e, ctx)
+
+
+@role.command('create', short_help='Creates role in the specified or current organization')
+@click.pass_context
+@click.argument('role-name',
+                metavar='<role-name>',
+                required=True)
+@click.argument('description',
+                metavar='<description>',
+                required=True)
+@click.argument('rights',
+                nargs=-1,
+                metavar='<rights>')
+@click.option('-o',
+              '--org',
+              'org_name',
+              required=False,
+              metavar='[org]',
+              help='name of the org',
+              )
+def create(ctx, role_name, description, rights, org_name):
+    try:
+        client = ctx.obj['client']
+        if (org_name is not None):
+            org_href = client.get_org_by_name(org_name).get('href')
+        else:
+            org_href = ctx.obj['profiles'].get('org_href')
+        org = Org(client, org_href)
```

Comment:
we could just pass 
Org(client, resource=client.get_org_by_name(org_name))

instead of passing org_href.

Reply:
fixed.